src/viam/examples/project/cmake/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 #   cmake -DCMAKE_PREFIX_PATH=$HOME/opt ...
 #
 if (NOT CMAKE_PREFIX_PATH)
-  set(CMAKE_PREFIX_PATH ${PROJECT_SOURCE_DIR}/../../../build/install)
+  set(CMAKE_PREFIX_PATH ${PROJECT_SOURCE_DIR}/../../../../../build/install)
```

Comment:
This isn't pretty but oh well. It wasn't pretty before. Now it is just 66% more displeasing.